.typos.toml

```diff
@@ -0,0 +1,9 @@
+# Copyright (C) The DDC development team, see COPYRIGHT.md file
+#
+# SPDX-License-Identifier: MIT
+
+[default.extend-words]
+# Don't correct these word (parts)
+benchs = "benchs"
```

Comment:
I'm not sure "benchs" is a word. Is this your abbreviation of benchmarks?

Reply:
It is a variable name in the splines benchmarks. I could also rename the variable.

Reply:
Up to you

Reply:
Renamed in 01584e9e45b458ee9d21dfdfab63681631dc278b